src/main/java/seedu/address/logic/commands/RepaidCommand.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.logic.commands;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.ReadOnlyPerson;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+
+/**
+ * 1. Adds a person identified using it's last displayed index into the whitelist.
```

Comment:
In my own opinion, the javadoc comments could be "Adds a person identified using it's last displayed index into the whitelist and resets the person's debt to zero." so that it fits the style of the rest of the javadoc comments.

Reply:
Noted.